package.json

```diff
@@ -5,13 +5,14 @@
   "devDependencies": {
     "grunt": "~0.4.2",
     "grunt-cli": "0.1.11",
-    "grunt-contrib-clean":"~0.6.0",
+    "grunt-contrib-clean": "~0.6.0",
     "grunt-contrib-jasmine": "^1.0.0",
     "grunt-contrib-sass": "0.7.4",
-    "jquery": "~1.11.3"
+    "jquery": "~1.11.3",
+    "standard": "^8.2.0"
```

Comment:
NB: Standard 8 was launched this month but `govuk_prototype_kit` for ex is still on Standard 7.

Updating is not really a priority IMO. (the only important changes are for JSX I think which we don't use)